### DISCUSSION TITLE
Open PRs

### DISCUSSION THREAD

**Discussion by bassstorm** [quodlibet/quodlibet#3655] [General]
Hi guys,

I see some pull requests are open since 2017. That gives probably not the best feeling for all parties.
I think it would be better to either close these or save to a branch in case there's a need to keep some code ideas (though it still makes little sense as it will go farther and farther from the master branch...).

Thanks for your opinions.

**Comment by lazka** [comment]
Our oldest still open bug report was in fact created in 2008 :D https://code.google.com/archive/p/quodlibet/issues/44

**Comment by declension** [comment]
Cleaned up a few
